src/lib/migration/main_workflow.rb

```diff
@@ -332,5 +347,34 @@ def init_pkg_mgmt
 
       ret
     end
+
+    # Enable the openSUSE => SUSE vendor change when migrating from
+    # an openSUSE distribution to SLE.
+    def vendor_init
+      File.write(VENDOR_FILE, VENDOR_CONTENT) if opensuse?
+    end
+
+    # Remove the vendor configuration file
+    def vendor_cleanup
+      File.delete(VENDOR_FILE) if File.exist?(VENDOR_FILE)
+    end
+
+    # Prepare the repositories for the online migration.
+    # This mainly activates some workarounds in the openSUSE => SLE migration
+    def prepare_repos
+      return unless opensuse?
+
+      # disable all enabled repositories, the migration only removes (upgrades)
+      # the repositories from the registration server (SCC), all other repositories
+      # (like the default OSS and non-OSS repos) would collide with the new SLES repos
```

Comment:
is this step documented somewhere for customers?

Reply:
It's documented in the manual migration via `zypper`/`SUSEConnect`: *"4. List and disable all openSUSE repositories on your system"* (https://susedoc.github.io/doc-sle/develop/SLES-upgrade/single-html/#sec.upgrade-online.opensuse_to_sle).

Reply:
ok